mergebot_poller_test.py

```diff
@@ -0,0 +1,109 @@
+import unittest
+import mergebot_poller
+from mock import patch
+
+
+class MergebotPollerTest(unittest.TestCase):
+
+    @patch('mergebot_poller.GithubPoller')
+    def testCreatePollerSuccess(self, mock_poller):
+        mergebot_poller.create_poller({'scm_type': 'github'})
+        assert mock_poller.call_count == 1
+
+    def testCreatePollerFailure(self):
+        self.assertRaises(AttributeError, mergebot_poller.create_poller,
+                          {'scm_type': 'unicorn'})
+
+    @patch('github_helper.GithubPR')
```

Comment:
This is really GithubPoller test. You don't have tests for MergebotPoller since it's an abstract class.